gcapi/gcapi.py

```diff
@@ -539,13 +537,47 @@ def __call__(
         else:
             return response
 
+    def run_external_job(
+        self, *, algorithm_name: str = None, inputs: Dict[str, any]
+    ):
+        algorithm = self._get_algorithm(algorithm_name=algorithm_name)
+        input_interfaces = {ci["title"]: ci for ci in algorithm["inputs"]}
+
+        for ci in input_interfaces:
+            if ci not in inputs and input_interfaces[ci]["default_value"] is None:
+                raise ValueError(f"{ci} is not provided")
+
+        job = {"algorithm_slug": algorithm["slug"], "inputs": []}
+        for input_title, value in inputs.items():
+            ci = input_interfaces.get(input_title, None)
+            if not ci:
+                raise ValueError(
+                    f"{input_title} is not an input interface for this algorithm"
+                )
+
+            i = {"interface": ci["slug"]}
+            if ci["kind"] in ("Image", "Segmentation", "Heatmap"):
```

Comment:
Maybe we need to expose whether this takes an image in the API as this could get stale.

Reply:
sorry, not sure what you mean :).

Reply:
SuperKind sounds very friendly :+1: , most inline with description of kind as well.